Docker run failed due to empty httpServerPort

**BUG REPORT**

***Describe the bug***
1. `BOOKIE_HTTP_PORT` is unset in `Dockerfile`.
2. `BK_httpServerPort` will be empty.
3. `httpServerPort` in `bk_server.conf` will be overrode to empty
4. Startup failed.

***To Reproduce***
1. Run image without additional env `BOOKIE_HTTP_PORT`.
2. Docker run failed.

***Expected behavior***
Only `BK_zkServers`(or other meta service related option) should be treated as must. All other should be optional for docker run.

***Screenshots***
```
2022-02-24 02:10:57,243 - INFO  - [main:Main@336] - Load lifecycle component : org.apache.bookkeeper.server.service.HttpService
Exception in thread "main" org.apache.commons.configuration.ConversionException: 'httpServerPort' doesn't map to an Integer object
        at org.apache.commons.configuration.AbstractConfiguration.getInteger(AbstractConfiguration.java:848)
        at org.apache.commons.configuration.AbstractConfiguration.getInt(AbstractConfiguration.java:822)
        at org.apache.bookkeeper.conf.ServerConfiguration.getHttpServerPort(ServerConfiguration.java:3347)
        at org.apache.bookkeeper.server.service.HttpService.publishInfo(HttpService.java:74)
        at org.apache.bookkeeper.common.component.LifecycleComponentStack.lambda$publishInfo$2(LifecycleComponentStack.java:130)
        at com.google.common.collect.ImmutableList.forEach(ImmutableList.java:405)
        at org.apache.bookkeeper.common.component.LifecycleComponentStack.publishInfo(LifecycleComponentStack.java:126)
        at org.apache.bookkeeper.common.component.ComponentStarter.startComponent(ComponentStarter.java:82)
        at org.apache.bookkeeper.server.Main.doMain(Main.java:234)
        at org.apache.bookkeeper.server.Main.main(Main.java:208)
Caused by: org.apache.commons.configuration.ConversionException: Could not convert  to java.lang.Integer
        at org.apache.commons.configuration.PropertyConverter.toNumber(PropertyConverter.java:461)
        at org.apache.commons.configuration.PropertyConverter.toInteger(PropertyConverter.java:294)
        at org.apache.commons.configuration.AbstractConfiguration.getInteger(AbstractConfiguration.java:844)
        ... 9 more
Caused by: java.lang.NumberFormatException: For input string: ""
        at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:65)
        at java.base/java.lang.Integer.parseInt(Integer.java:662)
        at java.base/java.lang.Integer.<init>(Integer.java:1105)
        at java.base/jdk.internal.reflect.GeneratedConstructorAccessor2.newInstance(Unknown Source)
        at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
        at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
        at org.apache.commons.configuration.PropertyConverter.toNumber(PropertyConverter.java:457)

```

***Additional context***
* Docker image `apache/bookkeeper:4.14.3`

***Possible fix***
Add `ENV BOOKIE_HTTP_PORT=8080` in `Dockerfile`

--- a/docker/Dockerfile
+++ b/docker/Dockerfile
@@ -25,7 +25,8 @@ ARG DISTRO_NAME=bookkeeper-server-${BK_VERSION}-bin
 ARG DISTRO_URL=https://archive.apache.org/dist/bookkeeper/bookkeeper-${BK_VERSION}/${DISTRO_NAME}.tar.gz
 
 ENV BOOKIE_PORT=3181
-EXPOSE $BOOKIE_PORT
+ENV BOOKIE_HTTP_PORT=8080
+EXPOSE $BOOKIE_PORT $BOOKIE_HTTP_PORT
 ENV BK_USER=bookkeeper
 ENV BK_HOME=/opt/bookkeeper
 ENV JAVA_HOME=/usr/lib/jvm/java-11

